test/boost/auth_resource_test.cc

```diff
@@ -10,11 +10,10 @@
 
 #include "auth/resource.hh"
 
-#include <sstream>
+#include <unordered_set>
 
 #include <boost/test/unit_test.hpp>
-
-#include "utils/to_string.hh"
+#include <fmt/ranges.h>
```

Comment:
we can drop `#include <utils/to_string.hh>`, as it provides the generic formatter  for `std::unordered_set<...>` which is not used anymore, once we have a better match of `std::unordered_set<resource>` defined in this change.